LoopUI/Views/LoopCompletionHUDView.swift

```diff
@@ -208,6 +222,50 @@ public final class LoopCompletionHUDView: BaseHUDView {
             accessibilityIdentifier = "loopCompletionHUDLoopStatusOpen"
         }
     }
+    
+    private func formattedTimeAgoString(_ timeString: String, includeGreaterThan: Bool = false) -> NSAttributedString {
+        let config = UIImage.SymbolConfiguration(pointSize: 10, weight: .semibold)
+        let symbol = UIImage(systemName: "arrow.trianglehead.2.clockwise", withConfiguration: config)
```

Comment:
I think you can use arrow.trianglehead.2.clockwise.rotate.90 instead of doing all the rendering and rotation.

Reply:
Thanks!